kirin/command/piv_worker.py

```diff
@@ -32,11 +32,12 @@
 
 
 from kirin import app, new_relic
+from kirin.core import model
```

Comment:
Minor, to shorten code a bit (and keep it clear):
To be done the same way everywhere IMO, so it may not be a good idea if `model` is used "from scratch" in a lot of places.
```suggestion
from kirin.core.model import Contributor
```

Reply:
Done in a81f9d8.

Reply:
Done (better) in 6689ed1.